meshtastic/admin.proto

```diff
@@ -279,3 +284,24 @@ message AdminMessage {
     int32 nodedb_reset = 100;
   }
 }
+
+/*
+ * Parameters for setting up Meshtastic for ameteur radio usage
+ */
+message HamParameters {
+  /*
+   * Amateur radio call sign, eg. KD2ABC
+   */
+  string call_sign = 1;
+
+  /*
+   * Transmit power in dBm at the LoRA transceiver, not including any amplification
+   */
+  int32 tx_power = 2;
+
+  /*
+   * The selected frequency of LoRA operation
+   * Please respect your local laws, regulations, and band plans.
+   */
+  float frequency = 3;
```

Comment:
will this replace the dedicated setting for tx power and frequency_override in device or is it an addition?

Reply:
This will set those lora settings in one operation, consistent for every client.